rad/src/hash_functions/mod.rs

```diff
@@ -1,13 +1,13 @@
 use std::fmt;
 
-use num_enum::TryFromPrimitive;
+use num_enum::{IntoPrimitive, TryFromPrimitive};
 
 use crate::error::RadError;
 use crate::hash_functions::sha2::sha2_256;
 
 mod sha2;
 
-#[derive(Debug, PartialEq, TryFromPrimitive)]
+#[derive(Debug, PartialEq, TryFromPrimitive, IntoPrimitive)]
```

Comment:
```suggestion
#[derive(Debug, PartialEq, IntoPrimitive, TryFromPrimitive)]
```

Reply:
I think is better to open an issue and refactor all of them